src/OpenTK/Math/Vector3d.cs

```diff
@@ -1221,14 +1411,6 @@ public static void CalculateAngle(ref Vector3d first, ref Vector3d second, out d
             return new Vector3d(v3.X, v3.Y, v3.Z);
         }
 
-        /// <summary>Converts OpenTK.Vector3d to OpenTK.Vector3.</summary>
```

Comment:
Just a question: Why delete this method?

Reply:
It was moved to OpenTK.Vector3